charts/spire/charts/spire-server/values.yaml

```diff
@@ -369,11 +365,34 @@ tornjak:
     successThreshold: 1
     timeoutSeconds: 5
 
+  # tornjak - Tornjak specific configuration
   config:
-    # -- persistent DB for storing Tornjak specific information
+    # -- Persistent DB for storing Tornjak specific information
     dataStore:
       driver: "sqlite3"
       file: "/run/spire/data/tornjak.sqlite3"
+
+    # -- Tornjak supports 3 connection types: `http`, `tls`, and `mtls`.
+    # Select only one.
+    connectionType: http
```

Comment:
I think we can drop this. Could be determined based on if there are tls certs or not.